### PR TITLE
chore: added cryptocurrency-app react project 

### PR DESCRIPTION
**Pull request by BhartiNagpure**
## Description
This is fully based on API fetching, in this you can find the different cryptoverse and their price as well as market cap and it also shows the graph for 24 hours, 3 Months , 30 Months and 1 Year.

## Checklist for projects addition 
 
- [x] My project has contribution guidelines.
- [x] My project has a Code of Conduct.
- [x] My project has a README file.
- [x] My project has a License.

---

Please note that if your project doesn't meet these requirements, your PR may be closed. Make sure to include these elements in your project before submitting the PR.

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -2873,5 +2873,23 @@
...
+  "Social_media": {
+    "gitHub": "https://github.com/BhartiNagpure",
+    "LinkedIn": "https://www.linkedin.com/in/nagpurebharti",
+    "Instagram": "https://instagram.com/bhartinagpure2409?igshid=MzNlNGNkZWQ4Mg==",
+    "YouTube": "#",
+    "Twitter": "#"
+  },
+  "Projects": [
+    {
+      "link": "https://bhartinagpure.github.io/Cryptocurrency/",
```
Comment (priyankarpal) [inline src/DB/projects.json:2888]: add the github repo link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
make sure that your project has contributing guidelines, license, readme, and code of conduct

**Review by BhartiNagpure — COMMENTED** [review]
added GitHub repo link

**Comment by BhartiNagpure** [comment]
> make sure that your project has contributing guidelines, license, readme, and code of conduct

@priyankarpal now my project has all of things which you mention above.

**Review by BhartiNagpure — COMMENTED** [review]
resolve conversation

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**